include/CppUTest/UtestMacros.h

```diff
@@ -249,8 +249,10 @@
 #define BITS_EQUAL_TEXT(expected, actual, mask, text)\
   BITS_LOCATION(expected, actual, mask, text, __FILE__, __LINE__)
 
+#define ROBUST_SIZEOF(type) (2*sizeof(type)/sizeof(short)) /* For the Cl2000, sizeof(char) == sizeof(short) == 1 */
```

Comment:
Hmmm. You know I don't like compiler specific code in the headers :) If we start that, it won't end eventually. Is there no way we can put this somewhere else or solve it another way?